docs/gui.rst

```diff
@@ -0,0 +1,40 @@
+..  _unpacking-gui:
+
+ReproUnzip GUI
+**************
+
+**reprounzip-qt** is a graphical interface (GUI) for reprounzip, allowing you to unpack and reproduce experiments from ``.rpz`` files without having to use the command-line. You can also set it as the default handler for the ``.rpz`` file extension so you can open them via double-click.
+
+Installation
+============
+
+*reprounzip-qt* comes with the installer on `Windows <https://github.com/ViDA-NYU/reprozip/releases/download/1.0.8/reprounzip-1.0.8-setup.exe>`_ and `Mac <https://github.com/ViDA-NYU/reprozip/releases/download/1.0.8/reprounzip-1.0.8.pkg>`_. If you used one of these, you will be able to double click on any ``.rpz`` file to boot up the GUI. 
+
+If you are using Anaconda, you can install *reprounzip-qt* from anaconda.org::
+
+    $ conda install --channel vida-nyu reprounzip-qt
+
+Otherwise, you will need to `install PyQt4 <https://www.riverbankcomputing.com/software/pyqt/download>`__ before you can install *reprounzip-qt* from pip (on Debian or Ubuntu, you can use ``apt-get install python-qt4``).
+
+On Linux, setting it as the default to open ``.rpz`` files is a bit more involved. Once the application is setup, run `this script <https://gist.github.com/remram44/0092c0b27269cfd0e5530428612d9309>`__ by  opening up a terminal window and entering ``bash register-linux.sh``. From there, you can back-click on a ``.rpz`` file, and set the default application to ReproUnzip. The following is an example of this process on Ubuntu 16.04: 
```

Comment:
I should probably put this script (and Windows equivalent?) into the reprounzip-qt code.